.github/workflows/ci.yml

```diff
@@ -3,38 +3,20 @@ on: [push, pull_request]
 permissions: {}
 
 jobs:
-  skipper:
-    runs-on: ubuntu-latest
-    permissions:
-      actions: write
-      contents: read
-    outputs:
-      should_skip: ${{ steps.skip_check.outputs.should_skip }}
-    steps:
-      - id: skip_check
-        uses: fkirc/skip-duplicate-actions@12aca0a884f6137d619d6a8a09fcc3406ced5281 # v5.3.0
-        with:
-          concurrent_skipping: 'same_content_newer'
-
   build:
-    needs: skipper
-    if: needs.skipper.outputs.should_skip != 'true'
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name
```

Comment:
Isn't this a bigger set than what is described in the PR? I would expect to have a condition like (event_name = push) || (different full names).

Reply:
I think the third option is the best/safest. I wouldn't count on github to never add new event type.

Reply:
It's not about github adding new type. It's about us having defined only those two at the top of the workflow file like this:
```
name: CI
on: [push, pull_request]
```

Reply:
But I'll update it so the intent is clear

Reply:
Fixed, I've also added trigger job back, so this huge `if` does not have to be repeated 5 times. Jobs that need other jobs are also cancelled if the ones they need are canceled.